ts/store/reducers/persistedPreferences.ts

```diff
@@ -32,6 +33,7 @@ export type PersistedPreferencesState = Readonly<{
   //       https://www.pivotaltracker.com/story/show/170998374
   isCustomEmailChannelEnabled: pot.Pot<boolean, undefined>;
   continueWithRootOrJailbreak?: boolean;
+  isMixpanelEnabled: boolean | null;
```

Comment:
I infer that `boolean | null` is used to capture three states: opt-in, opt-out, pristine. But I do have some context already.

I therefore suggest that we use a more specific type to better express our intentions to the uninformed reader, likely a self-explanatory union of strings.

Reply:
I'm agree with you. But using a nullable boolean is not a so bad choice:

`isMixpanelEnabled` -> `true` -> yes it is
`isMixpanelEnabled` -> `false` -> no it isn't
`isMixpanelEnabled` -> `null` -> no choice, we don't know if yes or not

For sure `"enabled" | "disabled" | "unknown"` could be better to express our intentions
Time is hard at this moment, I guess this could be a good compromise